src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    protected String name;
+    private Color color;
+
+    public Figure(String name, Color color) {
+        this.color = color;
+        this.name = name;
+    }
+
+    @Override
+    public String draw() {
+        return "Figure: " + name
+                + ", color: " + color
+                + ", area: " + (double) Math.round(getArea() * 100) / 100 + " sq. units, ";
```

Comment:
What is the reason to do: Math.round(getArea() * 100) / 100, why not just round?

Reply:
I changed it, but it would look better if we didn't see 100500 numbers after the dot.